<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1038</width>
    <height>618</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="Floor2Number">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>160</y>
      <width>101</width>
      <height>131</height>
     </rect>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>450</y>
      <width>1021</width>
      <height>156</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="1">
      <widget class="QPushButton" name="outfloor">
       <property name="text">
        <string>Уйти с этажа</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="Floor3Button">
       <property name="text">
        <string>3 этаж</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="Floor2Button">
       <property name="text">
        <string>2 этаж</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="Floor1Button">
       <property name="text">
        <string>1 этаж</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="outelevator">
       <property name="text">
        <string>Выйти из лифта</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="inelevator">
       <property name="text">
        <string>Зайти в лифт</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="onfloor">
       <property name="text">
        <string>Прийти на этаж</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="callElevator">
       <property name="text">
        <string>Вызвать лифт</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="Hint">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>20</y>
      <width>751</width>
      <height>401</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QLabel" name="Floor1Number">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>310</y>
      <width>101</width>
      <height>131</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QLabel" name="Floor3Number">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>0</y>
      <width>101</width>
      <height>131</height>
     </rect>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1038</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
